### ISSUE TITLE
Reset all positions foot differently

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/john-travoltage#312]
**Test device:**
Dell Laptop
**Operating System:**
Win 10
**Browser:**
Edge
**Problem description:**
For https://github.com/phetsims/QA/issues/196
When hitting the reset all button, John's foot ends up in slightly different positions.  Not exclusive to edge.
**Steps to reproduce:**
1. Move foot to new position.
2. Hit reset all.
3. Hit reset all a second time.  Note foot shift.

**Screenshots:**
![foot](https://user-images.githubusercontent.com/41024075/46155698-25eacc00-c235-11e8-91c4-5292dcff6ed7.gif)

**Troubleshooting information (do not edit):**

<details>

Name: ‪John Travoltage‬
URL: https://phet-dev.colorado.edu/html/john-travoltage/1.5.0-rc.1/phet/john-travoltage_en_phet.html
Version: 1.5.0-rc.1 2018-09-25 20:38:20 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36 Edge/17.17134
Language: en-US
Window: 1920x946
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0
Vendor: Microsoft (Microsoft Edge)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {"assert":{"sha":"928741cf","branch":"HEAD"},"axon":{"sha":"ea39b709","branch":"HEAD"},"brand":{"sha":"89d28f63","branch":"HEAD"},"chipper":{"sha":"ed169ed3","branch":"HEAD"},"dot":{"sha":"a6af5081","branch":"HEAD"},"john-travoltage":{"sha":"81c3b843","branch":"HEAD"},"joist":{"sha":"79cf90f9","branch":"HEAD"},"kite":{"sha":"a1257832","branch":"HEAD"},"phet-core":{"sha":"2ff7b489","branch":"HEAD"},"phet-io":{"sha":"c1301ac9","branch":"HEAD"},"phet-io-wrapper-classroom-activity":{"sha":"abc4ae3e","branch":"HEAD"},"phet-io-wrapper-hookes-law-energy":{"sha":"429886e7","branch":"HEAD"},"phet-io-wrapper-lab-book":{"sha":"186d07de","branch":"HEAD"},"phet-io-wrappers":{"sha":"9126d1ed","branch":"HEAD"},"phetcommon":{"sha":"33aef589","branch":"HEAD"},"query-string-machine":{"sha":"9ae63aab","branch":"HEAD"},"scenery":{"sha":"eb1234c9","branch":"HEAD"},"scenery-phet":{"sha":"d205340b","branch":"HEAD"},"sherpa":{"sha":"c6a137d3","branch":"HEAD"},"sun":{"sha":"6138c76e","branch":"HEAD"},"tambo":{"sha":"afb95743","branch":"HEAD"},"tandem":{"sha":"1dd1ec01","branch":"HEAD"},"vibe":{"sha":"82f7679d","branch":"HEAD"}}

</details>

**Comment by jessegreenberg** [comment]
This is likely related to #285, Ill look into this.

**Comment by KatieWoe** [comment]
Looks good on 1.5.0-rc.2
